### DISCUSSION TITLE
Following functionality for organizations

### DISCUSSION THREAD

**Discussion by ayoreis** [community/community#9934] [Other Feature Feedback, Questions, & Ideas]
It would be really nice to be able to "follow" / "watch" an organization as you can do with users!

**Comment by Gamebuster19901** [comment]
It seems this was implemented, but only partially. I cannot see when an organization creates a release, like I can with normal users.

**Reply by ayoreis** [reply]
Can organisations even create releases?

**Reply by Gamebuster19901** [reply]
A member of the organization can create a release in one of the organization's repositories.

It doesn't show up in either the user or organization feed.

**Reply by ayoreis** [reply]
~Don't really understand what you mean, but I get updates when a user I follow creates a new release on any repository.~ Ah ok, now I think I get it. So you mean since organizations can´t really do anything, that it should like follow the members actions inside that organization? Originally I requested this feature just for keeping track of organizations, though that would be cool as well, and showing how many follower a organization has too.
